fix: migrate customers table `current_url`/`pathname` fields from `string` to `text` types

kamilpikula opened the pull request:
### Problem

The current_url and pathname fields on the Customers table are sometimes longer than the max character limit for DB fields of type "string".

### Solution

Migrate the following fields on the Customers table from type "string" to type "text".
Created a migration script and updated this fields at the database level

### Note

`:text` type can be used in migration, but not the schema. In the database `:string` type will be mapped to `text` column type.

### Issue

[#328 ](https://github.com/papercups-io/papercups/issues/328)

## Checklist

- [x] Everything passes when running `mix test`
- [x] Ran `mix format`
- [x] No frontend compilation warnings

reichert621 reviewed (commented):
looks good @kamilpikula! will this affect any of our existing data?

also, do you mind adding some tests to `test/chat_api/customers_test.exs` where we create/update a Customer with a long (i.e. more than 255 char) string for these fields to verify that it works correctly?

thanks! 🎉

kamilpikula commented:
@reichert621 there shouldn't be any problems as we just changed the field type to `text` from `string`, so you can use more than 255 characters (limit of `string`)

I added the above fields to `update_customer/2` test and this seems to work. :)

reichert621 commented:
awesome, thanks @kamilpikula! 🎉
